react/Header/UserAccountMenu/UserAccountMenu.js

```diff
@@ -165,36 +216,8 @@ export default function UserAccountMenu({ locale, authenticationStatus, linkRend
         }
       </li>
       {
-        authenticationStatus === UNAUTHENTICATED ? (
-          <li className={`${styles.smallDeviceOnly}`}>
-            <span className={styles.item}>
-              {
-                linkRenderer({
-                  'data-analytics': 'sign-in',
-                  href: appendReturnUrl('/sign-in', returnUrl),
-                  className: styles.itemLink,
-                  title: 'Sign in',
-                  children: 'Sign in'
-                })
-              }
-              <span className={styles.secondaryItemText}>&nbsp;or&nbsp;</span>
-              {
-                linkRenderer({
-                  'data-analytics': 'register',
-                  href: appendReturnUrl('/sign-up', returnUrl),
-                  className: styles.itemLink,
-                  title: 'Register',
-                  children: 'Register'
-                })
-              }
-              <div className={styles.iconSpacer} />
-            </span>
-          </li>
-        ) : null
-      }
-      {
         authenticationStatus === AUTHENTICATED ? (
-          <li>
+          <li className={`${styles.aboveSmallDevice}`}>
```

Comment:
Is this class doing anything?

Reply:
Could we just replace it with Hidden component?

Reply:
Only reason was it's the current way the rest of the items in the menu where being hidden/shown - I guess it was done before `Hidden` component existed. I'll switch everything in this component over to `Hidden`.